src/main/java/bio/terra/cda/app/controller/QueryApiController.java

```diff
@@ -72,4 +74,25 @@ public QueryApiController(
 
     return sendQuery(querySql, offset, limit, dryRun);
   }
+
+  @Override
+  public ResponseEntity<QueryResponseData> uniqueValues(
+      String version, @Valid String body, @Valid Integer limit) {
+
+    String bqTable = "gdc-bq-sample";
+    Map<String, String> tableParts = CDAUtils.parseTableName(body);
+    logger.info("TABLE_PARTS " + tableParts.toString());
+    // SELECT * FROM UNNEST (SELECT DISTINCT p.column FROM qualifiedTABLE as p);
+    String querySql =
```

Comment:
I think it would make sense to add this code to `QueryTranslator`. Even though this isn't a query request, that code has unnest support and may be a good place to collect code that creates SQL queries.